src/CommandQueue.ts

```diff
@@ -47,6 +56,20 @@ export class CommandQueue {
         registerDomEvents: (payload: any): void => {
             this.orchestration.registerDomEvents(payload);
         },
+        recordEvent: (payload: RecordEventInput) => {
+            const type = payload.event_type;
+            const eventData = payload.event_data;
+            if (
+                type === undefined ||
+                eventData === undefined ||
+                typeof type !== 'string' ||
+                typeof eventData !== 'object'
```

Comment:
Can we make event data _not_ required?

Sometimes we may want to record a straight event with no data

Reply:
Updated CommandQueue to be able to send events without containing eventData field, and Orchestration to give default value of empty object when not passed in.